src/components/details/ProductDetail.jsx

```diff
@@ -0,0 +1,60 @@
+
+import ProductView from './ProductView'
+import Products from '../Products/Products';
+
+import './ProductDetail.scss'
+
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
Explore about the Context API from React

Reply:
replaced redux code with context api